docs/quickstarts/nextjs.mdx

```diff
@@ -177,26 +217,27 @@ export default function Home() {
 
 </CodeBlockTabs>
 
-### Sign out of your application
-
-Visit your Next.js application at [`localhost:3000`](http://localhost:3000), click on your avatar, and select **Sign out**. You are now back where you started: a sign-in form.
+Then, visit your app's homepage at [`localhost:3000`](http://localhost:3000) to see the `<UserButton />` in action. It should show your avatar from the account you signed in with.
 
-### Deploy your application
+### Sign out
 
-You're ready to [deploy your app to production](/docs/deployments/overview) and welcome new users!
+Finally, use the `<UserButton />` by selecting your avatar, and select **Sign out**. You should be redirected to the url you specified with `afterSignOutUrl`.
 
 </Steps>
 
-## You're authenticated!
+## Next steps
 
-Congratulations! Your app is now only available to authenticated users. But this is just the first step.
+Now try deploying your app to production:
 
-If you would like a repository that demonstrates many of the features Clerk has to offer, such as user, session, and organization management, check out one of Clerk's demo repositories:
+- See [Deploy to Vercel](/docs/deployments/deploy-to-vercel#deploying-a-clerk-app-to-vercel).
+- Or, if you're using another provider, see [Deploy to Production](/docs/deployments/overview).

```

Comment:
imo, i wouldn't even add this as a "next step", i would maybe add it as a card under "More resources".

Reply:
I haven't loved this as a last step. I don't think most devs would try deploying to production at that point. Having it as a card is probably a good play until the doc restructure takes shape.

Reply:
Great point! I've moved both these to cards under More Resources!